sdks/node/src/clientApi.ts

```diff
@@ -24,6 +24,9 @@ export async function login(apiKey: string, host?: string) {
     console.warn('No host provided, using default host:', defaultHost);
     host = defaultHost;
   }
+  if (!apiKey) {
```

Comment:
This is specifically for `apiKey == ''`?

Reply:
I think there was no validation in place to enforce this requirement before, but should raise an error now by adding that line. I tested with this script before:
```
import * as weave from 'weave';
async function main() {
    await weave.login()
    await weave.init('examples');
    console.log('Hello there, Weave JS');
}
main().catch(console.error);
```